pytorch_lightning/loops/batch/training_batch_loop.py

```diff
@@ -46,6 +40,7 @@ def __init__(self) -> None:
         # the current split index when the batch gets split into chunks in truncated backprop through time
         self.split_idx: Optional[int] = None
         self.optimizer_loop = OptimizerLoop()
+        self.manual_loop = ManualOptimization()
 
         self._warning_cache: WarningCache = WarningCache()
         self._hiddens: Optional[Tensor] = None
```

Comment:
I think now we can remove `_hiddens` in this class and instead track it inside the `OptimizerLoop` (already done) and `ManualOptimization`

Reply:
ah, seems like it!

Reply:
actually, could you double check again what you had in mind? the life cycle of hidden state here is actually tied to the training batch loop if I'm not mistaken. 

I don't see it at the moment

Reply:
You are right.